src/elixir/constants.ts

```diff
@@ -20,15 +20,15 @@ export const TICK_SPACINGS: { [amount in FeeAmount]: number } = {
   [FeeAmount.HIGH]: 200
 }
 
-export const POOL_INIT_CODE_HASH = '0x3fcff66860e53a77b6570a354f8e175e3e01fb76620272df5c59e6cf9bffc577'
+export const POOL_INIT_CODE_HASH = '0x41a723f9e6457830b1b7a44df4435fab88581d073226894b33131815dd674c22'
```

Comment:
@bilalcorbacioglu yes but this is not going to be changed in the chain where its already deployed. Because we are not going to deploy new contracts to those old chains.

Reply:
oh i see, this is elixir, nvm let me check